Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/PowerSupplies/SteamPowerSupply.cs

```diff
@@ -0,0 +1,129 @@
+﻿using Orts.Parsers.Msts;
+using ORTS.Scripting.Api;
+using System;
+using System.IO;
+
+namespace Orts.Simulation.RollingStocks.SubSystems.PowerSupplies
+{
+    /// <summary>
+    /// Basic power supply class for steam locomotives
+    /// For electrical systems powered by battery
+    /// </summary>
+    public class SteamPowerSupply : ILocomotivePowerSupply, ISubSystem<SteamPowerSupply>
```

Comment:
So, no scriptable steam locomotive power supply? @peternewell would be disappointed. :)

Reply:
Do you really have a complex electric power supply system on steam locomotives?
I don't think so! :D